### ISSUE TITLE
Ogma

### ISSUE THREAD

**Issue by titansystems** [dbarr/adesa#1]
not anywhere near a code issue.. just found this (as i lost the code in all my moves) and wondering if you are still watching.. would be nice to get back in touch.

**Comment by sheoncehadme** [comment]
well crap... didnt realize i was logged into a corporate account... hah...

**Comment by dbarr** [comment]
> not anywhere near a code issue.. just found this (as i lost the code in all my moves) and wondering if you are still watching.. would be nice to get back in touch.

Yep I'm still around! Just read the description of this repo: "Development stopped in 2005" and thought "holy crap that was 15 years ago". How time flies!

**Comment by sheoncehadme** [comment]
I know.. crazy this is that old! I kind of want to get the codebase back up and running for fun... not sure how to share info with each other via this public forum.. but definitely would like to catch up. I spent some time over the weekend trying to get it to compile but super hard to find all the correct old libraries to get it to go without doing some real work to update the code to use the current state of things..

**Comment by dbarr** [comment]
@sheoncehadme Added a who listing you might be interested in.
